tests/integration/kubernetes/gha-run.sh

```diff
@@ -236,6 +239,76 @@ function cleanup() {
 	kubectl delete -f "${tools_dir}/packaging/kata-deploy/kata-rbac/base/kata-rbac.yaml"
 }
 
+function deploy_snapshotter() {
+	echo "::group::Deploying ${SNAPSHOTTER}"
+
+	case ${SNAPSHOTTER} in
+		nydus) deploy_nydus_snapshotter ;;
+		*) >&2 echo "${SNAPSHOTTER} flavour is not supported"; exit 2 ;;
+	esac
+
+	echo "::endgroup::"
+}
+
+function deploy_nydus_snapshotter() {
+    echo "deploy_nydus_snapshotter"
+	NODE_NAME=${NODE_NAME:-$(hostname | tr A-Z a-z)} 
+    kubectl config set-context --current --namespace=default
+	nydus_snapshotter_version=$(get_from_kata_deps "externals.nydus-snapshotter.version")
+    sed -i -e "s|ghcr.io/containerd/nydus-snapshotter:latest|ghcr.io/containerd/nydus-snapshotter:${nydus_snapshotter_version}|g" "${NYDUS_SNAPSHOTTER_DEPLOY_DIR}/nydus-snapshotter.yaml"
+
+    sed -i -e "s|CONTAINERD_CONFI_DIR|${CONTAINERD_CONFI_DIR}|g" "${tools_dir}/tools/nydus-snapshotter.yaml"
```

Comment:
`${tools_dir}/tools/nydus-snapshotter.yaml` shoule be `${NYDUS_SNAPSHOTTER_DEPLOY_DIR}/nydus-snapshotter.yaml`?

I meet

Reply:
Fixed.